Vector/Model/Room/RoomDataSource.m

```diff
@@ -233,8 +233,31 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
                 bubbleCell.blurred = YES;
             }
         }
+
+        // Manage initial event (case of permalink or search result)
+        if (self.timeline.initialEventId)
+        {
+            // Reset the marker
+            if (bubbleCell.markerView)
```

Comment:
remove this marker systematically (do not limit this operation under "self.timeline.initialEventId != nil" condition)
A table view may dequeue a cell with a marker whereas its data source has been replace with a live data source...